### DISCUSSION TITLE
Exception thrown with PathToSwaggerGenerator set to empty string

### DISCUSSION THREAD

**Discussion by pacoteinnov** [Burgyn/MMLib.SwaggerForOcelot#237] [Q&A]
I would like that the JSON be available at https://server/api/v1/openapi.json instead of https://server/api/swagger/docs/v1/<swaggerkey>

In ocelot.json, I set the SwaggerKey to "openapi.json" and in UseSwaggerForOcelotUI options set PathToSwaggerGenerator to "" which makes the JSON available to https://server/api/v1/openapi.json but the UI gives me this exception :

```
System.Collections.Generic.KeyNotFoundException: The given key '/index.html' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at MMLib.SwaggerForOcelot.Repositories.SwaggerEndPointProvider.GetByKey(String key) in C:\r\MMLib.SwaggerForOcelot-swaggerforocelot_v5.2.0\src\MMLib.SwaggerForOcelot\Repositories\SwaggerEndPointProvider.cs:line 39
```

**Comment by Burgyn** [comment]
Hi @pacoteinnov,

please share `Ocelot.json` config for better investigation. Thansk.

**Reply by pacoteinnov** [reply]
Hi @Burgyn, I don't have the config anymore and we made it work by using https://server/api/openapi/docs/v1/openapi.json.
You can close this question, thanks!
